hack/make/resources/docker.exe.manifest

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
(so it's consistent with both `hack/make/.build-deb` and `hack/make/.build-rpm`)